src/headers.ts

```diff
@@ -41,7 +41,15 @@ const headerValueMappers = {
     })
       .filter(Boolean).join('; ')
   },
-  permissionsPolicy: (value: PermissionsPolicyValue) => Object.entries(value).map(([directive, sources]) => (sources as string[])?.length && `${directive}=${(sources as string[]).join(' ')}`).filter(Boolean).join(', ')
+  permissionsPolicy: (value: PermissionsPolicyValue) =>
+    Object.entries(value)
+      .map(
+        ([directive, sources]) =>
+          (sources as string[])?.length &&
```

Comment:
issue: here you can safely delete the checking for length as we will allow for empty arrays that will be translated to `()`